casa/OS/HostInfo.cc

```diff
@@ -124,7 +124,9 @@ Int HostInfo::numCPUs(bool use_aipsrc)				\
     static const String keyword("system.resources.cores");	\
     if ( use_aipsrc ) {						\
 	String value;						\
-	if ( Aipsrc::find(value, keyword) ) {			\
+	if (resources_cores > 0) {			\
+		return resources_cores;			\
```

Comment:
4 spaces should be the standard but it is very inconsistent, so either reindent the file correctly or use same indentation as the rest of the file